src/targets/index.ts

```diff
@@ -55,3 +56,9 @@ export function getTargetByName(
 ): typeof BaseTarget | undefined {
   return TARGET_MAP[targetName];
 }
+
+export function getTargetId(target: TargetConfig): string {
+  return target.id
+    ? `${target.id}[${target.name}]`
```

Comment:
For me personally it would be more intuitive to swap id and name, so we e.g. have `docker` as the generic target, and `docker[release]` or `docker#release` as more specific one.

Reply:
I agree and first I thought this would be a breaking change but it won't. I'll make that change and then merge.

Thanks for the feedback!